packages/css/src/style.ts

```diff
@@ -93,19 +93,37 @@ export function globalStyle(selector: string, rule: GlobalStyleRule) {
   appendCss({ type: 'global', selector, rule }, getFileScope());
 }
 
-export function fontFace(rule: FontFaceRule, debugId?: string) {
+export function fontFace(
+  rule: FontFaceRule | FontFaceRule[],
+  debugId?: string,
+) {
   const fontFamily = `"${cssesc(generateIdentifier(debugId), {
     quotes: 'double',
   })}"`;
 
+  const fontFamilyError = outdent`
+  This function creates and returns a hashed font-family name, so the "fontFamily" property should not be provided.
+
+  If you'd like to define a globally scoped custom font, you can use the "globalFontFace" function instead.
+`;
+
+  if (Array.isArray(rule)) {
```

Comment:
I think you can simplify this code by doing
```
const rules = Array.isArray(rule) ? rule : [rule];
```
and then just keeping one code path below.